extensions/resteasy-reactive/rest-client-reactive/runtime/src/main/java/io/quarkus/rest/client/reactive/runtime/RestClientCDIDelegateBuilder.java

```diff
@@ -104,6 +104,11 @@ private void configureCustomProperties(RestClientBuilder builder) {
         if ((headers != null) && !headers.isEmpty()) {
             builder.property(QuarkusRestClientProperties.STATIC_HEADERS, headers);
         }
+
+        if (configRoot.disableContextualErrorMessages != null) {
```

Comment:
And you don't need that test if you made it a `boolean`.

Reply:
Yeah, I still tend to make everything Optional. Fixed, thanks :).